.github/workflows/compile.yml

```diff
@@ -92,7 +95,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [
-#          ubuntu-20.04, # This is not built in llalam.cpp, so for now we'll skip it
+          ubuntu-22.04,
```

Comment:
I think this needs to remain 20.04. We moved down from 22.04 a while ago for compatibility reasons (https://github.com/SciSharp/LLamaSharp/commit/b52422a0280c638d1fd3f0f65b6eb90f261f4cc0).

Reply:
(Remember to change all of the `matrix.os == 'ubuntu-22.04` lines below)

Reply:
Unfortunately, in this case, 22.04 is required. As libclblast-dev does not exist prior to 22.04, at least not in the standard package repos.

https://packages.ubuntu.com/search?keywords=libclblast-dev&searchon=names

Reply:
Ok, that's not a major problem. It only affects the new binaries.